resources/stylesheets/inviwo.qss

```diff
@@ -82,21 +82,11 @@ QToolBar
 QToolBar {
     spacing: 0.1875em;
     padding: 0.0625em;
-    border: none;
-    border-bottom: 0.125em solid #2b2b2d;
-    padding-right: 0.5em;
     background-color: #323235;
-}
-QToolBar::handle {
```

Comment:
why were those removed?
![image](https://user-images.githubusercontent.com/9251300/157025492-9ab05436-d1b3-4885-8cbb-6152ce5b0f4f.png)

Reply:
I make them non movable, and non floatable. then that bar is not visible...